### ISSUE TITLE
Honour Craft's default site setting

### ISSUE THREAD

**Issue by russback** [verbb/navigation#245]
**What are you trying to do?**
I have 2 sites, the 2nd of which is set as the default site. I would like this site to be displayed as the selected site when a user goes to `/admin/navigation/navs/build/{id}` but the default site is not selected. Perhaps because it's selecting the first of the Sites returned from the Sites service.

**What's your proposed solution?**
SelectCraft's default site by default.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.14](https://github.com/verbb/navigation/releases/tag/1.4.14)
